README.md

```diff
@@ -40,7 +40,8 @@ Naga includes a default binary target, which allows to test the conversion of di
 cargo run --features wgsl-in -- my_shader.wgsl # validate only
 cargo run --features spv-in -- my_shader.spv my_shader.txt # dump the IR module into a file
 cargo run --features spv-in,msl-out -- my_shader.spv my_shader.metal --flow-dir flow-dir # convert the SPV to Metal, also dump the SPIR-V flow graph to `flow-dir`
-cargo run --features wgsl-in,glsl-out -- my_shader.wgsl my_shader.vert --profile es310 # convert the WGSL to GLSL vertex stage under ES 3.20 profile
+cargo run --features wgsl-in,glsl-out -- my_shader.wgsl my_shader.glsl.vert --profile es310 # convert the WGSL to GLSL vertex stage under ES 3.20 profile
+cargo run --features wgsl-in,hlsl-out -- my_shader.wgsl my_shader.hlsl.vert --shader-model 50 # convert the WGSL to HLSL vertex stage with shader model 5.0
```

Comment:
the features are no longer needed, the CLI binary now has them unconditionally

Reply:
I will remove all features requirements from README.